NOTES.md

```diff
@@ -1,3 +1,143 @@
+# Release 2.2.0 (Modern Times)
+
+Released on 2015-11-XX
+
+Release highlights:
+
+* Atomic changefeeds
+* Performance & Scalability improvements
+
+Read the [blog post][2.2-release] for more details.
```

Comment:
I think that this section needs to include one or two sentences describing what is exciting about this release, for example:

> RethinkDB 2.2 introduces atomic changefeeds, which can include existing values from the database into changefeed streams. These improved changefeeds makes building realtime apps dramatically easier: you can use a single code path to populate your application with initial data, and continue receiving realtime data updates.
>
> This release also includes a whole slew of performance and scalability improvements designed to help RethinkDB clusters scale to larger sizes while using fewer resources.